packages/events/EventPluginHub.js

```diff
@@ -189,27 +189,14 @@ export function extractEvents(
   return events;
 }
 
-/**
- * Enqueues a synthetic event that should be dispatched when
- * `processEventQueue` is invoked.
- *
- * @param {*} events An accumulation of synthetic events.
- * @internal
- */
-export function enqueueEvents(
-  events: Array<ReactSyntheticEvent> | ReactSyntheticEvent,
+export function runEventsInBatch(
```

Comment:
Happy to bikeshed on naming here. On the one hand it's not exactly clear this is flushing an *existing* queue as well. On the other hand, as far as I can tell, it's not supported to enqueue more events while we're extracting them (there's an invariant below saying it's not implemented). So the naming is probably fine since it's not supported to enqueue and *not* flush them anyway.

Reply:
I think it's fine, and much clearer.